apps/frontend/src/libs/components/sidebar/sidebar-items.tsx

```diff
@@ -0,0 +1,25 @@
+import { AppRoute } from "~/libs/enums/app-route.enum.js";
```

Comment:
same

Reply:
Just seen the same import in main branch in files \src\libs\components\protected-route\protected-route.tsx and \src\pages\not-found\not-found.tsx

Reply:
should be changed to "~/libs/enums/enums.js";

Reply:
i did it